packages/integration-tests/exec/build_and_run.sh

```diff
@@ -7,22 +7,17 @@ set -e
 
 cmd=$@
 
-ROOT_DIR=../../..
-
-
 if [ -n "$REBUILD" ]; then
-  echo -e "\n\nREBUILD env var set, rebuilding...\n\n"
+  echo -e "\nRebuilding integration tests\n"
 
   if [ -n "$FETCH_DEPS" ]; then
     echo -e "\nFetching dependencies (this will take forever the first time time)..."
-    yarn --cwd $ROOT_DIR --verbose --frozen-lockfile
```

Comment:
One thing to note is that the `ROOT_DIR` ends up being `/mnt` in the container, which is too deep. If we add other packages to this repo, we may have to update this script to ensure that it installs deps for each package.